host-spec/c03-transition.tm

```diff
@@ -626,26 +613,26 @@
 
   <subsection|Importing and Validating Block><label|sect-block-validation><label|sect-block-submission>
 
-  Block validation is the process by which the client asserts that a block is
-  fit to be added to the blockchain. This means that the block is consistent
-  with the world state and transitions from the state of the system to a new
-  valid state.
+  Block validation is the process by which a node asserts that a block is fit
+  to be added to the blockchain. This means that the block is consistent with
+  the world state and transitions from the current state of the system to a
```

Comment:
> world state

I know what you mean but this is a *interesting* term to use here. Might require clarification. Maybe just:

> This means that the block is consistent with the **current state and can transition to a new valid state**.

Reply:
It used the term "world state" also before my changes, but I agree that it probably should be changed, as we do not use this term anywhere else.

Reply:
You can blame this one on Syed.

Reply:
Fixed.